armotypes/vulnerabilitytypes.go

```diff
@@ -10,10 +10,16 @@ type VulnerabilityJobParams struct {
 }
 
 type Vulnerability struct {
```

Comment:
epss and Percentile are the only feilds from epss provider?

Reply:
we have also date but it's not on the design doc(new vulnerability table), do you want me to add it? do we have anything to do with that?
https://cyberarmor-io.atlassian.net/wiki/spaces/~631ef1f1f8c7bc1f3582e928/pages/290488544/Vulnerability+Management+Workload+Phase+1+-+Technical+Design#Notes-from-meeting-19%2F12%2F23

Reply:
let's add